config/config.exs

```diff
@@ -106,6 +106,10 @@ config :skate, Skate.Repo,
   port: System.get_env("POSTGRES_PORT", "5432") |> String.to_integer(),
   show_sensitive_data_on_connection_error: true
 
+config :skate, Skate.WarmUp,
+  minimum_percent_queries_to_succeed: 0.6,
```

Comment:
maybe this should be 100%? After testing two deploys, 100% of the queries succeeded on the first attempt.